linux_os/guide/services/ntp/service_ntpd_enabled/rule.yml

```diff
@@ -20,6 +20,9 @@ rationale: |-
 
 severity: medium
 
+identifiers:
+    cce@rhel7: CCE-83433-3
```

Comment:
@vojtapolasek Why is the CCE moved from gnome_gdm_disable_xdmcp? I would keep the existing CCE in gnome_gdm_disable_xdmcp and assign the new one here instead. Or is there some reason for this?

Reply:
Good point, this is a mistake. I will fix it.